config/canonical.json

```diff
@@ -14766,6 +14803,21 @@
       "name": "Cheesecake Factory"
     }
   },
+  "amenity/restaurant|Chevys Fresh Mex": {
+    "match": [
+      "amenity/fast_food|Chevys",
+      "amenity/restaurant|Chevys"
+    ],
+    "nocount": true,
+    "tags": {
+      "amenity": "restaurant",
+      "brand": "Chevys Fresh Mex",
+      "brand:wikidata": "Q5094466",
+      "brand:wikipedia": "en:Chevys Fresh Mex",
+      "cuisine": "mexican",
+      "name": "Chevys Fresh Mex"
```

Comment:
Is it actually used name? Maybe `name=Chevys`, `official_name=Chevys Fresh Mex` would be better? (Again I know nothing about this fast food).

Reply:
The name on the sign is Chevys Fresh Mex and the name that other maps use is Chevys Fresh Mex. It seemed appropriate.

Reply:
Personally id go with Chevys. Fresh Mex is another case of an unnecessary suffix. No one calls it that in the real world and there's almost zero chance of it being confused with another brand by naming it just Chevys. I agree it would work as an official name tag though (that's kind of what the point in the tag is).

Reply:
> Personally id go with Chevys. Fresh Mex is another case of an unnecessary suffix. No one calls it that in the real world and there's almost zero chance of it being confused with another brand by naming it just Chevys. I agree it would work as an official name tag though (that's kind of what the point in the tag is).

Yeah.. We have been inconsistent about this in the past too.. Let's just decide going forward to put the short "renderable" name in `name` and the longer name in `official_name`.   (looking at you "Applebee’s Neighborhood Grill & Bar" 👀 )

Reply:
I'll update this when I'm done with work and get Applebees in there as well. As long as it's available for search I'm fine with it. There's a bunch of them like this. Five Guys is really Five Guys Burgers and Fries, which should show up if you search for it, but doesn't need to display on the map. official_name should fix that nicely.

Reply:
@bhousel let me know if this is what you had in mind